tilleuls/sylius-click-n-collect-plugin/0.1/config/packages/sylius_click_n_collect.yaml

```diff
@@ -0,0 +1,3 @@
+imports:
+    # ...
+    - { resource: "@CoopTilleulsSyliusClickNCollectPlugin/Resources/config/app/config.yml" }
```

Comment:
Could you explain why you are importing this resource?

Reply:
I think here It's because of the [Sylius Grid Bundle](https://github.com/Sylius/SyliusGridBundle).

Reply:
Yes, and the SyliusUI Plugin. It's a common practice for Sylius plugins.